### PR TITLE
Animal database

### PR DESCRIPTION
**Pull request by davidigbo**
In this pull request I implement SQL vet_clinic database following these steps:

- Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- Create a database named vet_clinic.
- Create a table named animals with the following columns:
-  >d: integer
-  >name: string
-  >date_of_birth: date
-  >escape_attempts: integer
-  >neutered: boolean
-  >weight_kg: decimal
- Copy the SQL you used in the previous point into the schema.sql file in the generated repository.
 - Insert the following data:
-  >Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has - 
 >never tried to escape.
-  >Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
-  >Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
-  >Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Copy the SQL you used in the previous point into the data.sql file in the generated repository.
- Write queries for the following quests:
-  >Find all animals whose name ends in "mon".
-  >List the name of all animals born between 2016 and 2019.
-  >List the name of all animals that are neutered and have less than 3 escape attempts.
-  >List the date of birth of all animals named either "Agumon" or "Pikachu".
-  >List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

-Take a screenshot of the results of your queries.
-  [ ]image
-  [ ]
image

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @david30-maker ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Could you customize your readme file properly with your own information? Remember to remove the live demo section and also add 1 to 3 features to your Future Features section.
- [ ] You may also add screenshots for all queries to your pull request summary. 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by davidigbo** [comment]
Thanks @1GORDON for reviewing my project, I will implement the required changes

**Review by Davidosky007 — COMMENTED** [review]
Hi @david30-maker 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a generic readme template ,Kindly customize the readme to suit your project for example (project title, links should work etc) 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by davidigbo** [comment]
Thanks @Davidosky007 for checking my project

File: queries.sql [hunk 1]
```diff
@@ -1 +1,7 @@
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-13';
```
Comment (Shedrack-Sunday) [inline queries.sql]: - [ ] Kindly correct the `2019` date to cover all dates of the year. it should be `2019/31/12';` instead 👍

File: queries.sql [hunk 2]
```diff
@@ -1 +1,7 @@
...
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE name <> 'Gabumon';
```
Comment (Shedrack-Sunday) [inline queries.sql]: - [ ] You missed two major queries here. Find the requirement screenshot below, and implement accordingly 👍 

## Missed query:
- image
-  image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @david30-maker    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Moderate use of SQL queries 👍 

  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  features you missed

- In your README, you don't have a license file at the root, which is why your license file at the bottom is failing. kindly add the appropriate file path from the template provided 👍 

- You did not snap all the queries and include them in your PR. Seeing that you also missed two more queries, kindly make sure you add all screenshots of the query made as the next reviewer will check for those

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by davidigbo** [comment]
Thanks @Shedrack-Sunday for reviewing my project, I will implement the required changes

**Pull request merged**